kategory-test/src/main/kotlin/kategory/laws/OptionalLaws.kt

```diff
@@ -92,7 +92,7 @@ object OptionalLaws {
         override fun empty(): FirstOption<A> = FirstOption(Option.None)
 
         override fun combine(a: FirstOption<A>, b: FirstOption<A>): FirstOption<A> =
-                if (a.option.isDefined) a else b
```

Comment:
IMO these should not be internal but part of the public API, nothing wrong with asking an Option if it's defined, the `fold` method is too verbose for beginners and can be just as abused as here to obtain Boolean representations.